examples/keygen/keygen.c

```diff
@@ -69,9 +73,16 @@ int TPM2_Keygen_Example(void* userCtx, int argc, char *argv[])
             outputFile = argv[1];
     }
     while (argc > 1) {
+        if (XSTRNCMP(argv[argc-1], "-rsa", 4) == 0) {
+            alg = TPM_ALG_RSA;
+        }
         if (XSTRNCMP(argv[argc-1], "-ecc", 4) == 0) {
             alg = TPM_ALG_ECC;
         }
+        if (XSTRNCMP(argv[argc-1], "-sym", 4) == 0) {
```

Comment:
This is too generic. Can we expand this? There are several types of symmetric keys.

Reply:
we already have -aes for parameter encryption. Would a better name be -symaes ?

Reply:
How about `-sym=aesctr256`? And we can expand for other algos later.

Reply:
okay, makes sense. I will do it this way.